tardis/io/schemas/spectrum.yml

```diff
@@ -0,0 +1,18 @@
+type: object
+properties:
+  start:
+    type: quantity
+  stop:
+    type: quantity
+  num:
+    type: number
+    multipleOf: 1.0
+  method:
+    type: string
+    enum:
+            - integrated
+            - virtual
+            - real
+    default: virtual
+    description: The method to generate the final spectrum
+  description: Final spectrum sampling
```

Comment:
Very nice - for now we need to check however that FI is only used with:
- downbranch
- disable_electron_scattering